Makefile

```diff
@@ -1,7 +1,6 @@
-CC = gcc
-AR = ar
-override CFLAGS := -std=gnu99 -D_FILE_OFFSET_BITS=64 -Wall -Wextra -O3 -g -MMD $(CFLAGS)
-LDFLAGS =
+CC ?= gcc
+AR ?= ar
```

Comment:
Yes I know specifying the compilers etc. as arguments to `make` works, but env var doesn't work.

I'll remove the two lines shortly.